.github/actions/verify-dotnet-consolidation/action.yml

```diff
@@ -20,7 +20,8 @@ runs:
   steps:
     - name: Setup
       shell: pwsh
-      run: dotnet tool install dotnet-consolidate --global --version 2.0.0
+      run: |
+        & "${{ github.action_path }}/../../../Scripts/Install-DotNetTool.ps1" -Name "dotnet-consolidate" -Version "2.0.0" -Global
```

Comment:
Since Install-DotNetTool has a wider use-case it could be a good idea to turn it into a standalone action. That way it could be called even externally, or re-used in other actions of our own without referring to its physical location. (also using GHA parameters is nicer) What do you think?

Reply:
Good idea!

Reply:
I thought about that but wanted to get it working first. I'll test some further changes and get down to that.